core/src/main/java/com/github/mizool/core/configuration/Configuration.java

```diff
@@ -0,0 +1,8 @@
+package com.github.mizool.core.configuration;
+
+public interface Configuration
+{
+    String get(String key);
+
+    String get(String key, String def);
```

Comment:
Rename `def` to `default`. I think `System.getProperty(String key, String def)` was defined back in the day where the poor developers had to actually press keys for every single letter in their variable names and/or were not payed for "bytes of code produced".